xblock/core.py

```diff
@@ -174,6 +174,15 @@ def validate(self):
         """
         return Validation(self.scope_ids.usage_id)
 
+    def _(self, text):
```

Comment:
I'm wondering if we should use ugettext for this.  Also, please write a docstring that describes what this function actually does.